grid/communication/send-sms/maps/messente.test.ts

```diff
@@ -1,3 +1,3 @@
 import { sendSmsTest } from './send-sms';
 
-sendSmsTest('messente');
+sendSmsTest('messente', ['Superface', 'Superface']);
```

Comment:
Is not just one sender parameter enough?

Reply:
True